Octokit/Models/Request/NewTeamDiscussion.cs

```diff
@@ -0,0 +1,55 @@
+using System.Collections.ObjectModel;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    /// <summary>
+    /// Used to create a team discussion.
+    /// </summary>
+    /// <remarks>
+    /// <para>
+    /// In order to create a team discussions, the OAuth access tokens require the write:discussion scope.
+    /// </para>
+    /// <para>API: https://developer.github.com/v3/teams/discussions/#create-a-discussion </para>
+    /// </remarks>
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class NewTeamDiscussion
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="NewTeamDiscussion"/> class.
+        /// </summary>
+        /// <param name="title">The discussion post's title.</param>
+        /// <param name="body">The discussion post's body text.</param>
+        public NewTeamDiscussion(string title, string body)
+        {
+            Title = title;
+            Body = body;
+        }
+
+        /// <summary>
+        /// The discussion post's title (required).
+        /// </summary>
+        public string Title { get; private set; }
+
+        /// <summary>
+        /// The discussion post's body text (required).
+        /// </summary>
+        public string Body { get; set; }
+
+        /// <summary>
+        /// Private posts are only visible to team members, organization owners, and team maintainers.
+        /// Public posts are visible to all members of the organization. Set to true to create a private post.
+        /// Default: false.
+        /// </summary>
+        public bool Private { get; set; }
```

Comment:
Since this is an optional field it should probably be nullable (`bool? Private`) so that if the user doesn't set it, we dont send it to the API. Eventhough the default value is false anyway, there is a difference between us always sending false and us only sending a value when the user explicitly sets it.